src/main/scala/edu/ie3/simona/config/ConfigFailFast.scala

```diff
@@ -690,30 +704,74 @@ case object ConfigFailFast extends LazyLogging {
         s"There are multiple output configurations for participant types '${duplicateKeys.mkString(",")}'."
       )
 
+    implicit val participantNotifiers: Set[NotifierIdentifier.Value] =
+      NotifierIdentifier.getParticipantIdentifiers
+    configs.foreach(checkBaseOutputConfig)
+  }
+
+  /** Checks the thermal output configurations on duplicates
+    *
+    * @param configs
+    *   List of individual config entries
+    */
+  private def checkIndividualThermalOutputConfigs(
+      configs: List[SimonaConfig.BaseOutputConfig]
+  ): Unit = {
+    val duplicateKeys = configs
+      .map(config => StringUtils.cleanString(config.notifier).toLowerCase())
+      .groupMapReduce(identity)(_ => 1)(_ + _)
+      .filter { case (_, count) =>
+        count > 1
+      }
+      .keys
+
+    if (duplicateKeys.nonEmpty)
+      throw new InvalidConfigParameterException(
+        s"There are multiple output configurations for participant types '${duplicateKeys.mkString(",")}'."
+      )
+
+    implicit val thermalNotifiers: Set[NotifierIdentifier.Value] =
+      NotifierIdentifier.getThermalIdentifiers
```

Comment:
This contains some duplicates of `checkIndividualParticipantsOutputConfigs`. Maybe these can be joined to one method?